src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,65 @@
+import * as qs from "qs"
+import { data as sd } from "sharify"
+
+export const createCollectUrl = ({
+  dimension,
+  category,
+  artistId,
+}: {
+  dimension: "SMALL" | "MEDIUM" | "LARGE" | null
+  category: FilterCategory
+  artistId: string
+}) => {
+  let dimensionRange
+
+  if (dimension === "SMALL") {
+    dimensionRange = "0-1600"
```

Comment:
Steve just merged the PR to update vortex with proper size_score bands, so might as well fix these quickly before merging.

Reply:
Won't the dimension range stay the same?

Reply:
Ah yes, sorry. I was under the mistaken impression that dimension_range was based on size_score rather than width_cm and height_cm.